recipes/hostpython3/__init__.py

```diff
@@ -28,33 +28,11 @@ def prebuild_arch(self, arch):
         if self.has_marker("patched"):
             return
         self.copy_file("ModulesSetup", "Modules/Setup.local")
-        # self.apply_patch("ssize-t-max.patch")
-        # self.apply_patch("dynload.patch")
-        # self.apply_patch("static-_sqlite3.patch")
-        # shutil.copy("Modules/Setup.dist", "Modules/Setup")
-        # if "openssl.build_all" in self.ctx.state:
-        #     self.append_file("ModulesSetup.openssl", "Modules/Setup.local")
```

Comment:
I haven't played with kivy-ios much, but just curious why we have theses commented lines?
Wouldn't it be cleaner to just drop it?

Reply:
I don't understand, I already removed these lines.
Probably was some test that maybe @tito left over here last time he did a fix

Reply:
Yes I actually brain farted as I thought you added them :facepalm: 
I guess I hat too much of programming this weekend :smile: 
Thanks for the clean up :clap: